src/Symfony/Bridge/ProxyManager/Tests/LazyProxy/Fixtures/includes/foo.php

```diff
@@ -41,6 +41,16 @@ public function setBar($value = null)
         $this->bar = $value;
     }
 
+    public function __sleep()
```

Comment:
you'll need to add the `: array` return type to make tests pass

Reply:
done